components/classroom/groups/permissions_parameters.mjs

```diff
@@ -26,8 +26,8 @@ const Entity = {
     ALL: 'ALL'
 };
 
-module.exports = {
+/*module.exports = {
```

Comment:
we can just delete this instead of commenting it out

Reply:
Yes, could delete the commented things!